### PR TITLE
[Draft - please do not review yet] [DC-1899] [CircleCI] Run a nightly job to remove stale test buckets

### PR DESCRIPTION
**Pull request by hiro-mishima**

### CODE REVIEW

File: data_steward/tools/delete_stale_test_buckets.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+"""
+from google.cloud import storage
+from datetime import datetime, timezone
+import logging
+
+# Project imports
+from utils import pipeline_logging
+
+LOGGER = logging.getLogger(__name__)
+CLIENT = storage.Client(project='aou-res-curation-test')
```
Comment (MichaelSchmidt82) [inline data_steward/tools/delete_stale_test_buckets.py:12]: You can import `from gcloud.gcs import StorageClient` which will be scoped to this project. 🙂

Comment (hiro-mishima) [reply data_steward/tools/delete_stale_test_buckets.py:12]: Thanks, @MichaelSchmidt82 ! I wanted to avoid hardcoding the project, so I will try it 😃

**Review by MichaelSchmidt82 — APPROVED** [review]

**Comment by MichaelSchmidt82** [comment]
Solid design, solid work.

**Comment by hiro-mishima** [comment]
I will update this PR and enhance the scripts. I found some things need to be updated while working for the PR https://github.com/all-of-us/curation/pull/1038 .

**Comment by hiro-mishima** [comment]
@MichaelSchmidt82 I needed to rename my branch and that closed this pull request. (The same reason as https://github.com/all-of-us/curation/pull/1038#issuecomment-992970845). I will create another pull request and ask for your review again. Thanks!

**Pull request closed without merging**